packages/@react-aria/selection/src/ListKeyboardDelegate.ts

```diff
@@ -170,7 +170,7 @@ export class ListKeyboardDelegate<T> implements KeyboardDelegate {
     let key = this.collection.getFirstKey();
     while (key != null) {
       let item = this.collection.getItem(key);
-      if (item.type === 'item' && !this.disabledKeys.has(key)) {
+      if (item?.type === 'item' && !this.disabledKeys.has(key)) {
```

Comment:
Do we need this check in other methods like `getLastKey`, `getPreviousKey`, and `getNextKey` which have similar code?

Reply:
Well yes I agree, and that's why I [suggested](https://github.com/adobe/react-spectrum/issues/5680#issuecomment-1887307935) to convert the whole directory `packages/@react-aria/selection` to strict TS.

Maybe I should work on that next.